fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/Microsecond.java

```diff
@@ -50,6 +51,13 @@ public Microsecond(Expression arg) {
         super("microsecond", arg);
     }
 
+    @Override
+    public void checkLegalityBeforeTypeCoercion() {
+        if (!getArgument(0).getDataType().isDateTimeV2Type()) {
```

Comment:
if only support datetimev2 type, remove datev2 signature is just ok

Reply:
got it. fixed~